src/components/ConnectionPage/MyConnections.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import userPic from '../../assets/user.jpg'
+
+export interface MyConnectionsProps {
```

Comment:
Create a types file for these definitions so we can export/import them from an appropriate place, ie the `/types` dir.